conventions/src/main/kotlin/io.opentelemetry.instrumentation.javaagent-testing.gradle.kts

```diff
@@ -10,6 +10,16 @@ plugins {
   id("io.opentelemetry.instrumentation.javaagent-shadowing")
 }
 
+dependencies {
+  /*
+    Dependencies added to this configuration will be found by the muzzle gradle plugin during code
+    generation phase. These classes become part of the code that plugin inspects and traverses during
+    references collection phase.
+   */
+  add("codegen", "io.opentelemetry.javaagent:opentelemetry-javaagent-tooling")
```

Comment:
Just wondering: we already have this in the javaagent-instrumentation plugin, why do we need to duplicate it here?

Reply:
IIRC, which I might not :P, I think we said that we don't want this artifact on the muzzle check's transformation classpath. Is that not an issue anymore, or is that not what happens with this change?

Reply:
"This" - you mean "tooling"? `io.opentelemetry.instrumentation.javaagent-instrumentation.gradle.kts` always included tooling to `codegen` conf. Or do I miss something?

Reply:
Oh ok - previously for javaagent-testing.gradle.kts it was picking it up through the build.gradle dependencies but now both of them use the configuratio? Cool

Reply:
What I don't fully understand, is if we need muzzle-generation plugin at all in `io.opentelemetry.instrumentation.javaagent-testing.gradle`.

Reply:
At least one of our test projects includes a test instrumentation so it doesn't work without the codegen.